Classes/Sitegeist/MagicWand/Command/CloneCommandController.php

```diff
@@ -303,4 +304,16 @@ protected function checkConfiguration($remotePersistenceConfiguration)
         }
         $this->outputLine(' - Configuration seems ok ...');
     }
+
+    /**
+     * @return string
+     */
+    protected function getRsyncParams()
+    {
+        foreach ($this->rsyncParams as $index => $param) {
```

Comment:
Where is `$this->rsyncParams` coming from? Did you forget to `@InjectConfiguration` it?